keep-runtime/build.rs

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: Apache-2.0
+
+use std::fs;
+use std::path::{Path, PathBuf};
+
+fn main() {
+    let out_dir = std::env::var("OUT_DIR").unwrap();
+    let out_dir = Path::new(&out_dir).join("fixtures");
+    std::fs::create_dir_all(&out_dir).unwrap();
+
+    for entry in Path::new("fixtures").read_dir().unwrap() {
+        let wat = entry.unwrap().path();
+        if wat.extension().unwrap() == "wat" {
+            let wasm: PathBuf = Path::new(&out_dir)
+                .join(Path::new(wat.file_name().unwrap()))
+                .with_extension("wasm");
+
+            let bytes = wat::parse_file(&wat).unwrap();
+            fs::write(wasm, bytes).unwrap();
+
+            println!("cargo:rerun-if-changed={}", &wat.to_str().unwrap());
+        }
+    }
+}
```

Comment:
See below.

If we load the bytes using `wat::parse_file()` in the tests directly, then we don't need this build script at all.